service/src/main/java/dk/dbc/rr/oai/Config.java

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
this seems more or less a repetition of the config class from the formatter?

Reply:
Yup. Just feels like an overkill to put one class in it's own module. But there's only so many ways to extract config ;-)